src/cascadia/UIHelpers/ResourceString.cpp

```diff
@@ -0,0 +1,17 @@
+#include "pch.h"
+#include "ResourceString.h"
+#include "ResourceString.g.cpp"
+
+namespace winrt::Microsoft::Terminal::UI::implementation
+{
+    winrt::Windows::Foundation::IInspectable ResourceString::ProvideValue()
```

Comment:
we're not actually using this anywhere yet, right? just a "we can if we need it" thing? Maybe that's why we didn't merge this before?